showcase/src/app/router.config.ts

```diff
@@ -13,9 +13,9 @@ export function logRegisteredStates(registeredStates: StateDeclaration[]): void
 export function routerConfigFn(router: UIRouter): void {
 	router.trace.enable(Category.TRANSITION);
 	// Enable UI-Router visualizer here if needed (for development purposes only)
-	// if (ENV === "development") {
-	// 	router.plugin(Visualizer);  // Visualizer should be imported from "@uirouter/visualizer"
-	// }
+	if (ENV === "development") {
```

Comment:
This change should be reverted because we don't want to enable the visualizer by default.

Reply:
Yes of course, my bad. I've reverted the changes.